app/main/views/g_cloud.py

```diff
@@ -609,6 +609,38 @@ def why_didnt_you_award_contract(framework_framework, project_id):
     abort(404)
 
 
+@direct_award.route(
+    '/<string:framework_framework>/projects/<int:project_id>/tell-us-about-contract',
+    methods=['GET', 'POST']
+)
+def tell_us_about_contract(framework_framework, project_id):
+    all_frameworks = data_api_client.find_frameworks().get('frameworks')
+    framework = framework_helpers.get_latest_live_framework(all_frameworks, framework_framework)
+
+    # Get the requested Direct Award Project.
+    project = data_api_client.get_direct_award_project(project_id=project_id)['project']
+
+    form = TellUsAboutContractForm()
+
+    if form.validate_on_submit():
+        flash(Markup(f"""You've updated '{project['name']}'"""), 'success')
```

Comment:
I'd just go with double quotes unless you actually need the triple quotes, but I guess this is personal preference

Reply:
I went with triple quotes because all examples of flash messages I'd found had them, which is also why it is a Markup object. I agree it is unnecessary in this case but I wanted to be consistent.

Reply:
Looks like @samuelhwilliams beat me to this :) https://github.com/alphagov/digitalmarketplace-buyer-frontend/pull/763#discussion_r191420911

Reply:
I think it was probably someone copy-pasting a multiline one, and then it became an unstoppable meme, rather than an intentional consistency, but whatevs :)